flask_api.py

```diff
@@ -84,8 +85,14 @@ def handle_question():
     if "question" not in request_body:
         return "request body should include the question", BAD_REQUEST
 
-    response = {"answer": nimbus.answer_question(question)}
+    # Store
+    db = NimbusMySQLAlchemy(config_file=CONFIG_FILE_PATH)
```

Comment:
That makes sense. I think the whole ``flask_api.py`` is going to need a refactor since most routes init a database wrapper. I'll test it out with the global db and get a commit in soon

Reply:
@snekiam the function is now using the global singleton

Reply:
I think this is still initializing a new db object for each request.